integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -689,10 +691,16 @@ public int write(String path, Pointer buf, @size_t long size, @off_t long offset
       return -ErrorCodes.EEXIST();
     }
 
+    if (offset < oe.getWriteOffset()) {
```

Comment:
do you think it makes sense to support the case where FUSE writes to the range [0-100], then [50-150]? I don't know if FUSE would ever do this, but with the current code we'd ignore the full [50-150]. We could append only [100-150] instead.

Reply:
No, this should not be supported. The write to Alluxio must be sequential.

Reply:
sg